tests/integration_tests/adapters/interactive_brokers/conftest.py

```diff
@@ -110,23 +134,27 @@ def data_client(mocker, data_client_config, venue, loop, msgbus, cache, clock):
         ),
     )
     client = InteractiveBrokersLiveDataClientFactory.create(
-        loop=loop,
+        loop=event_loop,
         name=venue.value,
         config=data_client_config,
         msgbus=msgbus,
         cache=cache,
         clock=clock,
     )
-    client._client.start()
+    client._client._is_ib_connected.set()
+    client._client._connect = AsyncMock()
+    client._client._eclient = MagicMock()
+    client._client._account_ids = {"DU123456,"}
+    # client._client.start()
```

Comment:
Did we mean to leave these in? i.e. is the test still testing what we want it to?